dev-python/pipenv/metadata.xml

```diff
@@ -2,11 +2,19 @@
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
   <!--maintainer-needed-->
```

Comment:
Remove the comment. After this PR there is a maintainer for this, right? ;)

Reply:
Whoops :-) you are correct.